src/pages/grand-one.tsx

```diff
@@ -0,0 +1,495 @@
+              </Block>
+            </Block>
+          </Block>
+        </Block>
+      </Block>
+
+      <Block variant="div" id="section5">
+        <Block variant="div" className="container">
+          <Block mb="xl">
+            <Heading variant="h2">Laatu varmistetaan yhdessä oppien</Heading>
+          </Block>
+        </Block>
+        <Block variant="div" className="container half-container">
+          <Block variant="div" className="half">
+            <Paragraph marginBottomSpacing="l">
+              Suomi.fi Design Systemin tekemisen filosofiana on alusta asti
+              ollut pyrkimys korkeaan laatuun ja edelläkävijyyteen: on haluttu
+              näyttää hyvää esimerkkiä siitä, miten tehdä alusta asti
+              saavutettavaa Design Systemiä eikä vain tyydytä kopioimaan
+              valmiita ratkaisuja.
+            </Paragraph>
+            <Paragraph>
+              Jo konseptitason suunnitteluvaiheessa mukana on
+              saavutettavuusasiantuntija, joka myös validoi valmiit
+              ratkaisuehdotukset. Kaikille tekijöille pyritään saamaan myös
+              mahdollisimman hyvä saavutettavuusosaaminen, ja jokainen SFI DS:n
+              parissa työskentelevä on saanut saavutettavuuskoulutuksen. Tietoa
+              hyvistä ratkaisuista jaetaan aktiivisesti erilaisissa
+              keskeneräisten UX/UI-konseptien sparraussessioissa, joka toinen
+              viikko pidettävässä UX/UI-suunnittelutiimin yhteisessä
+              katselmoinnissa sekä Slackissa päivän polttavia kysymyksiä
+              ohjelmistokehittäjien ja UX/UI-suunnittelijoiden välillä
+              kiivaastikin pallotellen.
+            </Paragraph>
+          </Block>
+          <Block variant="div" className="half">
+            <Paragraph>
+              Tällä hetkellä SFI DS:n kehittämisestä vastaa neljän hengen
+              täysiaikainen ohjelmistokehittäjien tiimi. UX/UI-suunnittelu
+              toimii jaetun vastuun periaatteella DVV:n eri palveluihin
+              jalkautuneen 13 hengen konsulteista koostuvan suunnittelutiimin
+              kesken: jokainen saa osallistua SFI DS:n kehitykseen oman
+              mielenkiintonsa ja käytössä olevan ajan mukaan – toiset jo pitkään
+              ja vakiintuneina, toiset enemmänkin vierailevina tähtinä. Näin
+              jokaista komponenttia katsoo monta asiaansa vihkiytynyttä
+              silmäparia ja uudet komponentit, patternit ja muut elementit
+              välittyvät suunnittelijoiden mukana suorinta reittiä palveluihin.
+              Palveluista nousevia tarpeita puolestaan pallotellaan viikoittain
+              suunnittelijoiden ja SFI DS -tiimin ohjelmistokehittäjien kesken.
+            </Paragraph>
+          </Block>
+        </Block>
+        <Block variant="div" className="container">
+          <Block mt="xl">
+            <img
+              src={dsProcess}
+              alt="Suomi.fi Design Systemin kehittämisprosessi"
```

Comment:
I think this process should also be described in a way that is accessible to assistive technologies. Maybe visually hidden section with description of what's going on in the chart?